include/quda_milc_interface.h

```diff
@@ -660,9 +657,6 @@ extern "C" {
       double clover_coeff,
       QudaInvertArgs_t inv_args,
       const double* target_residual,
-      const void* milc_link,
```

Comment:
Will that break MILC?

Reply:
This was actually a bug fix, with inconsistent definition and declaration.  Clearly this hasn't been used in a long time.